R/p_map.R

```diff
@@ -41,7 +41,7 @@ print.p_map <- function(x, ...) {
 #' @export
 p_map.numeric <- function(posterior, precision = 2^10, ...) {
   # Highest density point
-  map <- map_estimate(posterior, precision = precision, density = TRUE)$MAP_density
+  map <- attributes(map_estimate(posterior, precision = precision))$MAP_density
```

Comment:
is `attributes` the correct function here? Shouldn't it rather be `map <- attr("MAP_density", map_estimate(posterior, precision = precision))$MAP_density)` or so?

Reply:
No, it should be:
`attr(map, "MAP_density") <- map_estimate(posterior, precision = precision)$MAP_density`

Reply:
No no, this is in `p_map` where I need to retrieve the density. The problematic part is in `map_estimate.R`